resources/assets/app.js

```diff
@@ -0,0 +1,21 @@
+var mymap = L.map('mapid');
+
+var osmUrl = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+var osmAttrib = 'Map data © <a href="http://openstreetmap.org">OpenStreetMap</a> contributors';
+
+var osm = new L.TileLayer(osmUrl, {
+  minZoom: 5,
+  maxZoom: 15,
+  attribution: osmAttrib
+});
+
+mymap.setView([-2, 120], 5);
+mymap.addLayer(osm);
+
+$(document).ready(function() {
+  $.getJSON('data/kalimantan-credits.json', function(data) {
+    $.each(data, function(name, count) {
```

Comment:
The list of names should be ordered.
I suggest highest contributors first, to lowest contributors.